src/Controllers/ImageController.php

```diff
@@ -72,6 +77,19 @@ public function __invoke(Request $request, string $size, string $file, string $w
         return $this->storage()->response($resizedPath);
     }
 
+    public function productImageUrlFromSku(string $sku): string
+    {
+        $productModel = config('rapidez.models.product');
+        $flat = (new $productModel())->getTable();
+
+        $product = $productModel::withoutGlobalScopes()
+            ->selectAttributes(['image'])
+            ->where($flat.'.sku', $sku)
```

Comment:
You could use [qualifyColumn](https://laravel.com/api/9.x/Illuminate/Database/Eloquent/Model.html#method_qualifyColumn) for this to be extra safe